### DISCUSSION TITLE
Why is there no ingress with min.io operator

### DISCUSSION THREAD

**Discussion by faxenmaxen** [minio/minio#11668] [Q&A] [answered]
Hello, 
why is there no ingress for the tenant-console?

my setup for e.g.
`$: kubectl minio init --namespace minio-operator`
`$: kubectl minio tenant create minio-tenant-1 --servers 4 --volumes 64 --capacity 100Gi --namespace minio-tenant-1 --storage-class local-storage`

```
$ kubectl -n minio-tenant-1 get pod,ep,svc,ing
NAME                                          READY   STATUS    RESTARTS   AGE
pod/minio-tenant-1-console-844bb97987-gpj45   1/1     Running   0          3d1h
pod/minio-tenant-1-console-844bb97987-qzrdv   1/1     Running   0          3d1h
pod/minio-tenant-1-ss-0-0                     1/1     Running   0          3d1h
pod/minio-tenant-1-ss-0-1                     1/1     Running   0          3d1h
pod/minio-tenant-1-ss-0-2                     1/1     Running   0          3d1h
pod/minio-tenant-1-ss-0-3                     1/1     Running   0          3d1h

NAME                               ENDPOINTS                                                   AGE
endpoints/minio                    10.0.0.88:9000,10.0.1.231:9000,10.0.4.73:9000 + 1 more...   3d1h
endpoints/minio-tenant-1-console   10.0.0.142:9443,10.0.6.157:9443                             3d1h
endpoints/minio-tenant-1-hl        10.0.0.88:9000,10.0.1.231:9000,10.0.4.73:9000 + 1 more...   3d1h

NAME                             TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
service/minio                    ClusterIP   10.111.15.120   <none>        443/TCP    3d1h
service/minio-tenant-1-console   ClusterIP   10.101.43.54    <none>        9443/TCP   3d1h
service/minio-tenant-1-hl        ClusterIP   None            <none>        9000/TCP   3d1h

```

**Comment by harshavardhana** [comment]
// cc @nitisht @dvaldivia

**Comment by dvaldivia** [comment, marked as answer]
hi @faxenmaxen , at the moment we don't configure any Ingress controller for every tenant as we haven't figure out a generic way to do so, we may expose some fields on the Tenant CRD in the future, the challenge with ingress is that everyone does it differently, some people like setting hardware load balancers that use node port services, in other cloud providers such as EKS you don't need an ingress controller, a simple LoadBalancer service will do, but on GCP ingress is needed if you want to use managed certificates, so that's why we didn't include an ingress configuration at the moment because we figured that everyone would expose their MinIO through the mechanism that makes sense for their deployment
